src/generator/__snapshots__/ssg.integration.js.snap

```diff
@@ -426,3 +426,94 @@ exports[`generates a simple playbook with uppercase profile name 1`] = `
       command: insights-client
       changed_when: false"
 `;
+
+exports[`ignores rsyslog_remote_loghost rules for compliance remediations 1`] = `
```

Comment:
Notice in this snap, the rule is simply skipped if supplied to the remediations API.